app/models/competition.rb

```diff
@@ -732,7 +732,7 @@ def uses_new_registration_service?
   end
 
   def should_render_register_v2?(user)
-    uses_new_registration_service? && user.cannot_register_for_competition_reasons(self).empty? && (registration_currently_open? || user_can_pre_register?(user))
```

Comment:
Why is this change necessary, what does it do in the context of the PR?

Reply:
Because before we showed the Registration Opens in x in Ruby only and now we want to show it in react